autosklearn/data/feature_validator.py

```diff
@@ -25,10 +25,8 @@
 class FeatureValidator(BaseEstimator):
     """
     Checks the input data to Auto-Sklearn.
-
```

Comment:
Please do not make such formatting changes. These will make it hard to merge later on.